### PR TITLE
Botany SOFT ranching opportunities. Cytology crate now orderable for botanists.

### PR DESCRIPTION
**Pull request by carshalash**
## About The Pull Request

image

Yeehaw, cowboy Zach here to bring ya the good news. Botanist will now be able to order a crate to set up cytology to grow some critters in botany. You can keep these lil critters around or give some FRESH TASTY produce to the kitchen so the crew isn't eating fucking monkey meat and impossible meat forever!

## Why It's Good For The Game

Honestly, I never see anyone use cytology outside of for the clown creatures. Hopefully, this PR it will encourage botanists to use it for either nutritional purposes or for fun. 

## Changelog

:cl:
expansion: Extends cytology to be an opportunity for botany through cargo interaction.
/:cl:

### CODE REVIEW

**Comment by Farquaar** [comment]
On one hand, generalizing cytology outside the realm of science is a big step.

On the other, I've noted several times about how hydroponics is much, much better suited to accomplishing cytology than xenobio. They have access to protein, and can grow many (but not most) of the reagents used in cytology. They lack maintenance access, but then again so do scientists. 

The only reason why cytology might conceptually be restricted to science is the availability of containment chambers, but if a botanist growing domestic animals shouldn't really need those (not like reinforced glass really holds back a space carp anyway).

In the end, this change doesn't seem too bad. It's gated by cargo, much in the same way that beekeeping is. It also opens up a ton of fun rancher gimmicks, which allow us to get more mileage out of those cowboy hats I added not too long ago.

TL;DR I've spent a long time thinking about the role of cytology, and this change makes a lot of sense.

**Comment by Farquaar** [comment]
I should add that one wants to restrict botanists from growing the more outlandish creatures (not sure if there's enough to warrant concern at this stage), I imagine one might make basic/farming biopsy tools and basic/farming swabs which would only pick up cell lines for more mundane creatures. That way, botanists get to be ranchers while the wild experiments tend to remain in xenobio.

**Comment by thgvr** [comment]
Goon ranching let's go

**Comment by Farquaar** [comment]
>thumbing down my farming biopsy tool idea
Answer for yourselves or return to your caves, you luddites!

**Comment by IndieanaJones** [comment]
I'll say that botany 100% is better suited to making mobs than xenobio is, since botany has the chems in-house whereas xenobio doesn't.  However, botany can't make as much use of them as xenobio can (essentially just the sentient slime potion)

**Comment by MrDoomBringer** [comment]
I thought the entire point of botany being good for cytology was to make botanists and scientists cooperate?

**Comment by Watermelon914** [comment]
Design doc for cytology suggests that it is for xenobiology, not botany.

Whilst I think botany may be better off with cytology content, you will need to make amendments to the design doc located here:
https://hackmd.io/@tgstation/rJXNUGwFL
Please get approval before committing any changes to the document

**Comment by carshalash** [comment]
Uhhh, the current plan is to find out how to make Petri dishes spawn with DNA. This crate will have cow and chicken petri dishes. (Maybe other animals in the future! Be optimistic) The tools to get more genetic samples will be gone, but the means to assemble cytology tubes and vats will remain.

To get more extreme DNA samples, botanists will need to work with scientists to get supplies.

**Comment by carshalash** [comment]
OKAY SO, a quick little update. This project has been PRETTY HEAVILY nerfed, botanists will no longer start with swabs or the biopsy tool, which means they will no longer be able to search out for samples themselves. This should quell people's fear that they will end up as 

The good news is, they will now be starting with petri dishes containing cow and chicken DNA (Thank you harse theef for showing me to how to do this!) The loadout of the cytology crate will now look like this.
image

This opens opportunities for having petri dishes orderable through cargo (or maybe even from the vending machine if it's popular enough!). I plan on making TWO whole new animals to the game soon they will be either be obtained through this crate or whole new ones.  

I will uh, probably be changing the cellulose requirement for cows to cooking oil to represent fat though, it is kinda weird that wood/cloth is required to make cows. Just need permission first 🥺

**Comment by optimumtact** [comment]
this doesn't make much sense given cytology is a job for the xenobiologist.

there's no backing for this.

**Comment by bruhlookatthisdood** [comment]
just go steal from xenobio

**Comment by optimumtact** [comment]
also there was an entire actual ranching thing that goof started work on, why not revive that.
